docs/panels.rst

```diff
@@ -189,6 +189,17 @@ line_profiler_.
 
 .. _line_profiler: https://github.com/rkern/line_profiler
 
+LDAP Tracing
+~~~~~~~~~~~~
+
+URL: https://github.com/danyi1212/django-windowsauth
+
+Path: ``windows_auth.panels.LDAPPanel``
+
+LDAP Operations performed during the request, including timing, request and response messages, the entries recieved, write changes list, stacktracing and error debugging.
```

Comment:
This is one of the lines causing the lint test to fail. Try running `make style` or shorten the lines here. Line 201 may also need it.

Please change recieved to received.

Reply:
Oh totally missed that 😅 Thank you!